components/list/list-item-mixin.js

```diff
@@ -125,6 +125,14 @@ export const ListItemMixin = superclass => class extends composeMixins(
 				position: absolute;
 				width: 100%;
 			}
+            :host([_show-add-button][_has-nested-list]) [slot="control-container"]::after {
+                background-clip: content-box, border-box;
+                background-image: linear-gradient(white, white), linear-gradient(to right, var(--d2l-color-mica) 20%, transparent 20%, transparent 80%, var(--d2l-color-mica) 80%);
+                background-origin: border-box;
+            }
+			:host([_show-add-button][_has-nested-list]:not([selection-disabled]):not([skeleton])[selected]) [slot="control-container"]::after {
+                background-image: linear-gradient(white, white), linear-gradient(to right, #b6cbe8 20%, transparent 20%, transparent 80%, #b6cbe8 80%);
+			}
```

Comment:
This is for the case where there's a nested list and we want to show the first part of the border.
We could actually remove the end part being mica as well, that was just leftover from the gradient below.